src/components/layout/index.tsx

```diff
@@ -31,6 +32,11 @@ const useStyles = makeStyles({
 export const Layout = ({ children }: any) => {
   const { account, activate } = useWeb3React<Web3Provider>()
   const { content, contentWrap } = useStyles()
+  const { web3Account, setWeb3Account } = useStore()
+
+  useEffect(() => {
```

Comment:
is it necessary to do it there? write console.logs in every useEffect for this purpose and check how much they are